src/ac_copilot_trainer/modules/hud.lua

```
@@ -1,62 +1,34 @@
-local COLOR_BG        = rgbm(0.067, 0.067, 0.067, 0.60)
-local COLOR_BG_BORDER = rgbm(0.30, 0.32, 0.38, 0.40)
-local COLOR_RED       = rgbm(0.937, 0.267, 0.267, 1.0)   -- #EF4444
-local COLOR_WHITE     = rgbm(0.949, 0.949, 0.960, 1.0)   -- #F2F2F5
-local COLOR_AMBER     = rgbm(1.000, 0.769, 0.239, 1.0)   -- #FFC43D
-local COLOR_LABEL_GREY = rgbm(0.549, 0.565, 0.612, 1.0)  -- #8C909C
-local COLOR_BRAND_GREY = rgbm(0.435, 0.459, 0.522, 1.0)  -- #6F7585
-local COLOR_TITLE     = rgbm(0.35, 0.82, 0.95, 1.0)      -- cyan (legacy path)
+local COLOR_BG_DARK   = rgbm(17 / 255, 17 / 255, 17 / 255, 0.60)
+local COLOR_BG_BORDER = rgbm(239 / 255, 68 / 255, 68 / 255, 0.40)  -- red-500/40
+local COLOR_RED       = rgbm(239 / 255, 68 / 255, 68 / 255, 1.0)   -- #EF4444 (per spec)
+local COLOR_RED_LIGHT = rgbm(248 / 255, 113 / 255, 113 / 255, 1.0) -- red-400 (alt title)
```

Luacheck (1.2.0)

[warning] unused variable 'COLOR_RED_LIGHT'

(W211)